http/api/searchProducts.ts

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios';
+import { ProductResult } from '../../@types';
+
+const MARKETPLACE_URL = `https://coral-app-8bk8j.ondigitalocean.app/api/product-retrieval/`;
+const axiosSearchInstance = axios.create({
```

Comment:
you guys don't have an instance already defined?

Reply:
there is in the axios.ts file. used it and got cors policy error so i created one in the meantime.

Reply:
alright